Gazebo/models/wsc_aircraft/model.sdf

```diff
@@ -0,0 +1,1803 @@
+<?xml version="1.0" ?>
+<sdf version="1.9">
+  <model name="wsc_aircraft">
+  <static>0</static>
+  <self_collide>0</self_collide>
+
+    <!-- Links -->
+
+    <link name="base_link">
+      <inertial>
+        <pose>0 0 0 0 0 0</pose>
+        <mass>0.001</mass>
+        <inertia>
+          <ixx>1e-10</ixx>
+          <ixy>0</ixy>
+          <iyy>1e-10</iyy>
+          <ixz>0</ixz>
+          <iyz>0</iyz>
+          <izz>1e-10</izz>
+        </inertia>
+      </inertial>
+
+      <!-- centre of mass debugging 
+      <collision name="base_link_collision">
+        <pose>0 -0.78 0 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>0.1 0.1 2</size>
+          </box>
+        </geometry>   
+      </collision>
+
+      -->
+
+    <!-- save for debugging use -->
+    <!--
+      <visual name="cp_wing_root_left">
+        <pose>1.7 -0.15 2.15 0 0 0</pose>
+        <geometry>
+          <sphere>
+            <radius>0.03</radius>
+          </sphere>
+        </geometry>
+        <material>
+          <ambient>0 0 1</ambient>
+          <diffuse>0 0 1</diffuse>
+          <specular>0.1 0.1 0.1 1</specular>
+        </material>
+      </visual>
+
+      <visual name="cp_wing_tip_left">
+        <pose>3.75 0.6 2.2 0 0 0</pose>
+        <geometry>
+          <sphere>
+            <radius>0.03</radius>
+          </sphere>
+        </geometry>
+        <material>
+          <ambient>0 0 1</ambient>
+          <diffuse>0 0 1</diffuse>
+          <specular>0.1 0.1 0.1 1</specular>
+        </material>
+      </visual>
+    -->
+    </link>
+
+    <link name="wing_body">
+      <pose>0 -1.47 1.95 0 0 0</pose>
+      <inertial>
+        <mass>0.001</mass>
+        <inertia>
+          <ixx>1e-10</ixx>
+          <ixy>0</ixy>
+          <iyy>1e-10</iyy>
+          <ixz>0</ixz>
+          <iyz>0</iyz>
+          <izz>1e-10</izz>
+        </inertia>
+      </inertial>
+    </link>
+
+<link name="wing_root_left">
+      <pose>-0.1 0 0 -0.05 0.05 0</pose>
+      <inertial>
+        <pose>1.28 -0.576 2 0 0 0</pose>
+        <mass>9.3</mass>
+        <inertia>
+          <ixx>3.159</ixx>
+          <ixy>-2.957</ixy>
+          <iyy>6.887</iyy>
+          <ixz>0.067</ixz>
+          <iyz>0.082</iyz>
+          <izz>9.962</izz>
+        </inertia>
+      </inertial>
+      <visual name="wing_root_left_visual">
+        <geometry>
+          <mesh>
+            <uri>model://wsc_aircraft/meshes/wing_root_left.dae</uri>
+          </mesh>
+        </geometry>
+        <material>
+          <ambient>0 1 0</ambient>
+          <diffuse>0 1 0</diffuse>
+          <specular>0.1 0.1 0.1 1</specular>
+        </material>
+      </visual>
+      <collision name="wing_root_left_collision">
+        <geometry>
+          <mesh>
+            <uri>model://wsc_aircraft/meshes/wing_root_left.dae</uri>

```

Comment:
This and other review comments are 'would be nice to fix' for future PRs but aren't blockers for this one.
  
May get better physics performance using custom collision meshes. These can either be primitives, or mesh files created by applying a decimation modifier to the original collada mesh. STL format for collisions result in smaller file sizes as it does not carry texture details.